teps/0084-endtoend-provenance-collection.md

```diff
@@ -188,13 +129,349 @@ The `in-toto` format created by chains wraps the [SLSA Provenance v0.2](https://
   * **results**: An array of objects representing the results of a `taskrun`. Each object contains the attribute `name` and `value`. Their values are of type string. `taskrun` results are important in the context of a `pipelinerun` because `taskrun` results may be used by other tasks, thus impacting the output of the `pipelinerun`.
 * **predicate.materials**: For `taskruns`, Chains populates this object based on specially named task parameters or results, i.e. `CHAINS-GIT_COMMIT` and `CHAINS-GIT_URL`. Similarly, Chains must populate this object with the corresponding pipeline parameters or results.
 
+<details>
+    <summary>Click to view an example pipelinerun attestation</summary>
+
+```json
+{
+  "_type": "https://in-toto.io/Statement/v0.1",
+  "predicateType": "https://slsa.dev/provenance/v0.2",
+  "subject": [
+    {
+      "name": "registry.example.com/minimal-container/min",
+      "digest": {
+        "sha256": "52af519cd2cb835e77f971ab303a8ae1c3b9b04fe2aa604d1c50d8d73f110304"
+      }
+    }
+  ],
+  "predicate": {
+    "builder": {
+      "id": "https://tekton.dev/chains/v2"
+    },
+    "buildType": "https://tekton.dev/attestations/chains/pipelinerun@v2",
```

Comment:
Yeah, I am trying to set the `buildType` field using the run object's GroupVersionKind i.e. `tekton.dev/v1beta1/kind=TaskRun` in https://github.com/tektoncd/chains/pull/495.

So for pipelinerun, I guess it will be `tekton.dev/v1beta1/kind=PipelineRun`?